website/pages/docs/agent/options.mdx

```diff
@@ -1815,6 +1821,10 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
 
 - `retry_interval_wan` Equivalent to the [`-retry-interval-wan` command-line flag](#_retry_interval_wan).
 
+- `rpc.enable_streaming` - Enable the gRPC subscribe endpoint on a Consul Server. All
```

Comment:
Do we use the `<outer>.<inner>` syntax anywhere else on this page. If a user tried to put that directly in the hcl or json I don't think it would parse so it might be worth the extra effort to create the nested configuration under the outer `rpc` stanza